jbpm-flow/src/main/java/org/jbpm/workflow/instance/impl/WorkflowProcessInstanceImpl.java

```diff
@@ -409,12 +410,17 @@ public void start(String trigger) {
 			registerExternalEventNodeListeners();
 			// activate timer event sub processes
 	        Node[] nodes = getNodeContainer().getNodes();
+	        // OCRAM: reverse order for stackless
 	        for (Node node : nodes) {
 	            if (node instanceof EventSubProcessNode) {
 	                Map<Timer, DroolsAction> timers = ((EventSubProcessNode) node).getTimers();
 	                if (timers != null && !timers.isEmpty()) {
-	                    EventSubProcessNodeInstance eventSubprocess = (EventSubProcessNodeInstance) getNodeInstance(node);
-	                    eventSubprocess.trigger(null, org.jbpm.workflow.core.Node.CONNECTION_DEFAULT_TYPE);
+	                    EventSubProcessNodeInstance eventSubprocess = (EventSubProcessNodeInstance) createNodeInstance(node);
+	                    if( isStackless() ) {
```

Comment:
same here, could eventSubprocess.trigger hide this check and decision if it stackless or not